tests/integration/test_consumer.py

```diff
@@ -108,7 +110,7 @@ def test_consumer(producer, consumer_queue: Queue, session):
     producer.enqueue_message(json.dumps(_raw_metrics).encode("utf-8"), _query_id)
 
     while (_message_key := consumer_queue.get().key()).decode("utf-8") != _query_id:
-        logging.debug("Got _message_key=%s which is not _query_id=%s", _message_key, _query_id)
+        logger.debug("Got _message_key=%s which is not _query_id=%s", _message_key, _query_id)
```

Comment:
i noticed the usage of f-strings in prior loggers. is there a reason to use both %formatting and f-strings

Reply:
ideally, we should use % formatting for performance reasons, the logger will check if that level needs to be logged and only the call `str(obj)` and splice the string. so for a log like this one, if the log level is INFO or WARNING it will not use resources to format the string.
if we were to use f string the logger would not have the chance to make optimizations

in theory using f strings for error and warning logs should no be a problem since these will be logged in most cases, but for consistency we should change them to % formatting